### PR TITLE
Localization support

### PR DESCRIPTION
**Pull request by VincentLoy**
Hi ! :ok_hand: 

Localization support, based on PR #21.
This is an huge update of the code.

- Resolve conflicts between PR #21 and master
- Add `.gitignore` file
- Split code into small functions
- Improve indent (using 2 spaces)
- Replace `.txt` locale files extension by `.json`
- Store all locale data in locale json files
    - `tag_prefix`
    - `separator_regex`
    - `tag_ends_with`
    - `tag_ends_with_map`
    - `emoji_map`
- Use `fetch` API instead of XHR, wait for `localeData` to be loaded before launching page scraping.
- Add french support :fr:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (jtojnar) [inline .gitignore:1]: This should probably be in your global gitignore, it is not feasible for projects to keep track of every editor or development environment.

File: manifest.json [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
   "name": "Show Facebook Computer Vision Tags",
   "version": "0.0.1",
+  "author": "Adam Geitgey",
   "manifest_version": 2,
   "description": "Shows what Facebook thinks your pictures contain",
   "permissions": [
     "http://www.facebook.com/*", "https://www.facebook.com/*"
   ],
+  "web_accessible_resources": [ "locales/*"],
+  "browser_action": {
```
Comment (jtojnar) [inline manifest.json:11]: The button does nothing but take space, that’s why it was removed.

File: locales/en_US/messages.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{
+  "tag_prefix": "Image may contain: ",
+  "separator_regex": ", | and ",
+  "tag_ends_with": "people",
+  "tag_ends_with_map": "2 people",
+  "emoji_map": {
+    "1 person": "👤",
+    "2 people": "👥",
+    "3 people": "👥",
```
Comment (jtojnar) [inline locales/en_US/messages.json:9]: Not necessary thanks to `tag_ends_with`.

**Comment by ageitgey** [comment]
Thanks! This works for me!

File: facebook.js [hunk 4]
```diff
@@ -115,14 +43,52 @@ const show_facebook_cv_tags = function() {
...
+};
 
-show_facebook_cv_tags();
+
+/**
+ * Make the fetch request to get locale data
+ * @param localePath
+ */
+const makeLocaleRequest = function (localePath) {
+  fetch(chrome.extension.getURL(localePath)).then(function (response) {
```
Comment (jtojnar) [inline facebook.js:69]: Simplify:
```javascript
  fetch(chrome.extension.getURL(localePath))
    .then(response => response.json())
    .then(data => initializePlugin(data))
    .catch(function (err) {
      console.error('FB COMPUTER VISION TAGS ERROR', err);
    });
```

**Comment by jtojnar** [comment]
Looks like I was too slow.

**Comment by ageitgey** [comment]
Sorry @jtojnar, but you made good points! Happy to take a PR to make those changes.

**Comment by jtojnar** [comment]
Won’t have time to do it today. If anyone else can do it, feel free.

File: facebook.js (hunk 4 above)
Comment (VincentLoy) [reply facebook.js:69]: Nice point, I'll check for update it

File: locales/en_US/messages.json (hunk 3 above)
Comment (VincentLoy) [reply locales/en_US/messages.json:9]: You right 👍

File: manifest.json (hunk 2 above)
Comment (VincentLoy) [reply manifest.json:11]: You right, this is a fail when fixing conflits with PR #21. :face_with_head_bandage:

**Pull request merged**